src/main/webapp/careerquiz.js

```diff
@@ -0,0 +1,76 @@
+/*
+* Purpose: recieves HTTP promise response from Career Quiz servlet, displays questions and responses 
+*/
+
+function getQuestionsAndChoices() {
+ const responsePromise = fetch('/careerquiz');
+ responsePromise.then(handleResponse);
+}
+
+function handleResponse(response) {
+ const responsePromise = response.json();
+ responsePromise.then(addToDom);
```

Comment:
this and `getQuestionsAndChoices` seems to only have a 1 space indentation

Reply:
changed.